modules/lazyload/assets/js/frontend.js

```diff
@@ -0,0 +1,22 @@
+document.addEventListener( 'DOMContentLoaded', function() {
+    const dataAttribute = 'data-e-bg-lazyload-src';
+    const lazyloadBackgrounds = document.querySelectorAll( `[${ dataAttribute }]:not(.lazyloaded)` );
+    const lazyloadBackgroundObserver = new IntersectionObserver( ( entries ) => {
+        entries.forEach( ( entry ) => {
+            if ( entry.isIntersecting ) {
+                let [ lazyloadBackground, element ] = [ entry.target, entry.target ];
+                const lazyloadSelector = lazyloadBackground.getAttribute( 'data-e-bg-lazyload-selector' );
```

Comment:
maybe instead of having multiple data attrs, use a single one as `JSON.stringify`?